pom.xml

```diff
@@ -228,7 +228,7 @@
       <dependency>
         <groupId>com.fasterxml.jackson.core</groupId>
         <artifactId>jackson-databind</artifactId>
-        <version>${jackson.version}</version>
+        <version>${jackson.version}.2</version>
```

Comment:
Q: Will also releases have a .2?  e.g. will there never be a 1.0.0 etc?

Reply:
Just this one, I expect it to break on upgrade resulting in this `.2` being removed

Reply:
Cool maybe have a comment to that effect to prevent people from just asking you at that time (ala gitblame).